AutocompleteClient/FW/Logic/Request/CIOQuizQuery.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CIOQuizQuery.swift
+//  Constructor.io
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+/**
+ Struct encapsulating the necessary and additional parameters required to execute a quiz query.
+ */
+public struct CIOQuizQuery: CIORequestData {
+    /**
+     The id of the quiz
+     */
+    public let quizId: String
+
+    /**
+     A list of answers that be in the following formats:
+     * Single Select: "answer1"
+     * Multiple Select: "answer1, answer2"
+     * Open Text: "true"
+     * Cover Page: "seen"
+     
+     Please refer to "https://docs.constructor.io/rest_api/quiz/using_quizzes/" for additional details
+     */
+    public var answers: [String]?
+
+    func url(with baseURL: String) -> String {
+        return "" // Do nothing - Returns empty string to satsify protocol requirement
+    }
+
+    func urlWithFormat(baseURL: String, format: String) -> String {
```

Comment:
Not sure if this is the best way to handle this, but this will make it so the consumer of the client doesn't have to pass in `finalize` as an argument when calling `getQuizResults`.